res/values/attrs.xml

```diff
@@ -11,4 +11,18 @@
 
 	</declare-styleable>
 
+	<string-array name="pref_behaviour_entries">
+		<item>do nothing</item>
```

Comment:
I think it is better to define a string per item and use them in the string-array item instead of using a literal. 
Besides, we are using Transifex to translate into different languages and it takes the file strings.xml to do the translation.